api/types/accesslist/member.go

```diff
@@ -99,15 +99,7 @@ func (a *AccessListMember) CheckAndSetDefaults() error {
 	}
 
 	if a.Spec.Name == "" {
-		return trace.BadParameter("member name is missing")
-	}
-
-	if a.Spec.Joined.IsZero() || a.Spec.Joined.Unix() == 0 {
-		return trace.BadParameter("member %s: joined field empty or missing", a.Spec.Name)
-	}
-
-	if a.Spec.AddedBy == "" {
-		return trace.BadParameter("member %s: added_by field is empty", a.Spec.Name)
```

Comment:
Just to confirm, Joined and AddedBy are always set the grpc Endpoint during access list creatoin so there is no need to fill those filed in the client side ?

Reply:
Yes `populateMemberFields` [does that](https://github.com/gravitational/teleport.e/blob/dc8b64cfa77f06ff6a49835b8bc11ccf1953f27d/lib/accesslist/service.go#L1011-L1031). `populateMemberFields` is called in `upsertAccessListWithMembers` [here](https://github.com/gravitational/teleport.e/blob/dc8b64cfa77f06ff6a49835b8bc11ccf1953f27d/lib/accesslist/service.go#L1492) and `upsertAccessListMember` [here](https://github.com/gravitational/teleport.e/blob/dc8b64cfa77f06ff6a49835b8bc11ccf1953f27d/lib/accesslist/service.go#L958)